src/themes/light.scss

```diff
@@ -107,7 +107,7 @@
 
 // temporary logic for legacy
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-light {
-  .wxc-banner.my-preview { display: flex; }
+  .wxc-interstitial-meeting .wxc-banner { display: flex; }
```

Comment:
this will be removed when the themes are ready, display: flex needs to be still present after this

Reply:
the banner is already displayed flex